### PR TITLE
improve preact

### PR DESCRIPTION
**Pull request by samccone**
If a user has an element with an ID of `c` on the page... they get a free game of breakout.

### CODE REVIEW

File: src/preact.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { render } from './render';
...
1!f
2|<f
!(=||(	
	0einnerText(d,e)=>e%?d:"\\n"+d))},;10
0]	e].fill(=d.map(]*=-1)|(
(||3,c.	d[
b[
)&&/|0,0)]=e=nter=(+';for(Y in $='

	')with(_.split($[Y]))_=join(pop());eval(_)
```
Comment (developit) [inline src/preact.js]: unnecessary spaces here are doubling the size of this otherwise extremely useful addition

Comment (samccone) [reply src/preact.js]: unicode characters !== Whitespace.

**Comment by notwaldorf** [comment]
LGTM

**Comment by k1r0s** [comment]
image

File: src/preact.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { render } from './render';
...
+
```
Comment (NekR) [inline src/preact.js]: Please formatting in a separate PR.

**Comment by NekR** [comment]
@samccone it's too hard to be a contributor, yeah.

**Comment by ashinzekene** [comment]
😪😪😪

**Pull request closed without merging**